core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -685,7 +685,10 @@ private[ui] class TaskDataSource(
 
   private var _tasksToShow: Seq[TaskData] = null
 
-  override def dataSize: Int = taskCount(stage)
+  override def dataSize: Int = {
+    val storedTasks = store.taskCount(stage.stageId, stage.attemptId).toInt
```

Comment:
You don't need the intermediate variable here.

Reply:
Done. Thanks